docs/pages/release_notes.rst

```diff
@@ -5,6 +5,46 @@ Release notes
 
 Whats has been changed, and how to update between versions.
 
-******
-v1.0.0
-******
+***************
+Almond (v1.0.0)
+***************
+
+Release date: 2022-04-01
+
+This is the initial release of the Nuts node reference implementation.
+It implements RFC001 - RFC017 specified by the `Nuts specification <https://nuts-foundation.gitbook.io>`_.
+This release is intended for developers. It contains a stable API that will be backwards compatible for the next versions.
+The releases until the first production release will mainly focus on network and OPS related features.
+
+To start using this release, please consult the getting started section.
```

Comment:
Not sure if this is supposed to be in release notes, but perhaps a link to read the docs here?

Reply:
these notes are on read the docs...